### ISSUE TITLE
Crouton 3.11.10 broken

### ISSUE THREAD

**Issue by EddieW1** [bmlt-enabled/crouton#309]
Update todayna.org to Crouton 3.11.10 and the site stopped displaying meetings.

**Comment by radius314** [comment]
I hit one bug on a map search on 

```
crouton.nojquery.min.js?ver=1607219791:1 Uncaught TypeError: Cannot read property 'split' of undefined
    at Object.<anonymous> (crouton.nojquery.min.js?ver=1607219791:1)
    at Object.eval [as main] (eval at createFunctionContext (crouton.nojquery.min.js?ver=1607219791:1), <anonymous>:5:122)
    at c (crouton.nojquery.min.js?ver=1607219791:1)
    at r (crouton.nojquery.min.js?ver=1607219791:1)
    at s (crouton.nojquery.min.js?ver=1607219791:1)
    at Object.t.invokePartial (crouton.nojquery.min.js?ver=1607219791:1)
    at Object.invokePartial (crouton.nojquery.min.js?ver=1607219791:1)
    at 1 (crouton.nojquery.min.js?ver=1607219791:1)
    at s (crouton.nojquery.min.js?ver=1607219791:1)
    at p (crouton.nojquery.min.js?ver=1607219791:1)
```

**Comment by radius314** [comment]
happens only with text search, may or may not be related

**Comment by radius314** [comment]
likely unrelated

**Comment by radius314** [comment]
related to #353

**Comment by EddieW1** [comment]
Here are the short codes we use on Todayna.org:

The Southern California Region has [group_count service_body_parent="1073"] Groups, hosting [meeting_count service_body_parent="1073"] Meetings a month.

[bmlt_tabs service_body_parent="1073"]

**Comment by radius314** [comment]
fix for this is coming in 3.11.11

**Comment by JonasServicekonto** [comment]
The shorcode on the Danish page is this:

[bmlt_tabs int_start_day_id="2" header="1" time_format="HH:mm" language="da-DK" has_areas='1' has_cities='1' has_groups='0' has_locations='0' custom_query="&formats=-58"]
